llvm/lib/DebugInfo/DWARF/DWARFDebugLine.cpp

```diff
@@ -1477,10 +1497,11 @@ bool DWARFDebugLine::Prologue::getFileNameByIndex(
 }
 
 bool DWARFDebugLine::LineTable::getFileLineInfoForAddress(
-    object::SectionedAddress Address, const char *CompDir,
+    object::SectionedAddress Address, const char *CompDir, bool ApproximateLine,
     FileLineInfoKind Kind, DILineInfo &Result) const {
   // Get the index of row we're looking for in the line table.
-  uint32_t RowIndex = lookupAddress(Address);
+  auto RowIndexValue = lookupAddress(Address, ApproximateLine);
```

Comment:
Maybe don't use auto and/or use a more descriptive variable name - it's not clear here what this is/what the first/second are (a named structure with named members, as suggested elsewhere, might help too)

Reply:
Yes, removed the usage of auto also ```lookupAddress``` return type will now be the same as in previous case i.e ```uint32_t```. Remove the usage of std::pair data structure.